wsmaster/che-core-api-metrics/pom.xml

```diff
@@ -59,6 +59,10 @@
             <groupId>org.eclipse.che.core</groupId>
             <artifactId>che-core-api-workspace-shared</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.eclipse.che.infrastructure</groupId>
```

Comment:
why do we need this dependency?

Reply:
because of new `CurrentLogwatchersMeterBinder` that watches events from `infrastructure-metrics` module

Reply:
I'm not sure if `infrastructure-metrics` module added any useful value here. I would put it to `infrastructure-kubernetes`.

Reply:
done

Reply:
-1

Reply:
api should not depend on infrastructure implementation.

Reply:
ok, reverted last commit